<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>531</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>HelloRadio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QDial" name="volumeD">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>80</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="volumeL">
             <property name="text">
              <string>VOL</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="volumeLCD">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>14</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="digitCount">
              <number>3</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="value" stdset="0">
              <double>80.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="playPB">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>108</width>
             <height>108</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/visual/play-button.svg</normaloff>:/visual/play-button.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="jsConsolePB">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Console</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/visual/console.svg</normaloff>:/visual/console.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="aboutQtPB">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>About Qt</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/visual/information.svg</normaloff>:/visual/information.svg</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="num1PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="num2PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="num3PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>3</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="num4PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="num5PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="num6PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>6</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="num7PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>7</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="num8PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>8</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="num9PB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>9</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="otherPB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="urlLE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>http://us2.internet-radio.com:8443/stream</string>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jsConsoleGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="javaScriptL">
          <property name="text">
           <string>JavaScript Console</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="runJsPB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>RUN</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/visual/run-script.svg</normaloff>:/visual/run-script.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="jsPTE">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string>alert('hello world');</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>runJsPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>runJavaScript()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jsPTE</sender>
   <signal>textChanged()</signal>
   <receiver>MainWidget</receiver>
   <slot>javaScriptChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeD</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWidget</receiver>
   <slot>volume(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWidget</receiver>
   <slot>urlChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jsConsolePB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>toggleJsConsole()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>834</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num1PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num2PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num3PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num4PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num5PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num6PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num7PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num8PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num9PB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setPresetStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>setCustomStation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutQtPB</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>aboutQtInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runJavaScript()</slot>
  <slot>javaScriptChanged()</slot>
  <slot>urlChanged(QString)</slot>
  <slot>play()</slot>
  <slot>stop()</slot>
  <slot>cors()</slot>
  <slot>volume(int)</slot>
  <slot>toggleJsConsole()</slot>
  <slot>setPresetStation()</slot>
  <slot>setCustomStation()</slot>
  <slot>aboutQtInfo()</slot>
 </slots>
</ui>
